src/callbacks_step/glm_speed.jl

```diff
@@ -52,21 +53,18 @@ end
 # This method is called as callback after the StepsizeCallback during the time integration.
 @inline function (glm_speed_callback::GlmSpeedCallback)(integrator)
 
-  dt = integrator.dtcache
+  dt = get_proposed_dt(integrator)
   semi = integrator.p
-  mesh, equations, solver, cache = mesh_equations_solver_cache(semi)
-  @unpack glm_scale, cfl_scale = glm_speed_callback
+  _, equations, solver, cache = mesh_equations_solver_cache(semi)
```

Comment:
```suggestion
  mesh, equations, solver, cache = mesh_equations_solver_cache(semi)
```
See explanation below